### PR TITLE
New fetch function to access eZSiteData in templates

### PR DESCRIPTION
**Pull request by pkamps**
New fetch function allows template developers to fetch content from the ezsite_data table. Here is an example:

{fetch( 'content', 'site_data', hash( 'name', 'ezpublish-version' ) )|dump()}

### CODE REVIEW

File: kernel/content/function_definition.php [hunk 1]
```diff
@@ -1222,4 +1222,21 @@
                                                    'parameter_type' => 'standard',
                                                    'parameters' => array() );
 
+$FunctionList[ 'site_data' ] = array(
```
Comment (peterkeung) [inline kernel/content/function_definition.php]: This needs to follow the same code formatting as all of the definitions above

Comment (pkamps) [inline kernel/content/function_definition.php]: fixed

File: kernel/content/function_definition.php [hunk 2]
```diff
@@ -1222,4 +1222,13 @@
                                                    'parameter_type' => 'standard',
                                                    'parameters' => array() );
 
+$FunctionList[ 'site_data' ] = array( 'name' => 'site_data',
```
Comment (peterkeung) [inline kernel/content/function_definition.php]: This is picky, but there should not be extra spaces in the square brackets

**Comment by pkamps** [comment]
spacing from square brackets removed

**Comment by peterkeung** [comment]
+1

**Comment by benkmugo** [comment]
+1 go for it

**Pull request merged**